docs/index.md

```diff
@@ -3,17 +3,10 @@ layout: page
 title: AddressBook Level-3
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2223S1-CS2103T-T13-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2223S1-CS2103T-T13-1/tp/actions)
 [![codecov](https://codecov.io/gh/AY2223S1-CS2103T-T13-1/tp/branch/master/graph/badge.svg?token=GB1YKZLVSX)](https://codecov.io/gh/AY2223S1-CS2103T-T13-1/tp)
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
-
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
-
-
```

Comment:
Can we keep this as reference until we want to replace it with our own introduction? Doesn’t make sense to delete it

Reply:
ok